Example/RxGesture/ViewController.swift

```diff
@@ -194,7 +216,6 @@ class ViewController: UIViewController {
         view.rx
             .longPressGesture()
             .when(.began)
-            .observeOn(MainScheduler.asyncInstance)
```

Comment:
Is this now delivering on MainScheduler by default ? Wondering why it was removed.

Reply:
If I remember correctly, I removed it because it was just not necessary at the first place.